src/foraging_gui/Foraging.py

```diff
@@ -1196,7 +1194,8 @@ def _GetSettings(self):
         for key in defaults:
             if key not in self.Settings:
                 self.Settings[key] = defaults[key]
-                logging.warning('Missing setting ({}), using default: {}'.format(key,self.Settings[key]))
+                logging.warning('Missing setting ({}), using default: {}'.format(key,self.Settings[key]),
```

Comment:
I think I would leave these off the warning box, since the RAs dont know what they mean